### PR TITLE
KR: #357, #359, #360 and more translations

### PR DESCRIPTION
**Pull request by telk5093**
Applied #357, #359, #360 as @NeverDruid did in #365.
And translated more strings and fixed some typos.

### CODE REVIEW

File: data/language/korean.txt [hunk 1]
```diff
@@ -11255,7 +11257,7 @@ STR_SCNR    :유럽 - 혁신
 STR_PARK    :잿더미를 딛고
 STR_DTLS    :방치되어 있는 낡은 공원이 있습니다. 이 공원이 과거에 누렸던 영광을 재현하기 위해 유럽 연합의 허가를 받았습니다! 허가를 받은 만큼 공원을 혁신해서 보답하세요.
 
-<N America - Extreme Hawaiian Island>
+<N. America - Extreme Hawaiian Island>
```
Comment (Gymnasiast) [inline data/language/korean.txt]: Change this back. This is a filename, which you shouldn't change.

Comment (telk5093) [inline data/language/korean.txt]: But it does not be translated if this dot is missed.

I attached that the image using this korean.txt language file
image

Comment (Gymnasiast) [inline data/language/korean.txt]: Hm, that's strange. In that case you can leave it as it is.

Comment (telk5093) [inline data/language/korean.txt]: Isn't it the scenario's name, not the file name?

Comment (Gymnasiast) [inline data/language/korean.txt]: Probably. It must have been changed with the introduction of the new scenario dialog.

**Pull request merged**